# Documentation for the PPARgene database


Peroxisome proliferator-activated receptors (PPARs) are members of nuclear receptor / ligand-activated transcription factors superfamily. PPAR subfamily consists of 3 subtypes: PPAR-α, -β/δ, and -γ. PPARs form heterodimers with a retinoid X receptor (RXR) and bind to specific PPAR-responsive elements (PPREs) to regulate target gene expression. In the absence of specific ligands, the PPAR-RXR heterodimer forms repressive complex with co-repressors and histone deacetylases. Upon ligand binding, the receptors undergo conformational changes that cause the dissociation of repressors, the recruitment of coactivators, and the activation of gene transcription. 

The PPARs play critical physiological roles as lipid sensors and regulators of glucose/lipid metabolism. Synthetic PPAR ligands, including the fibrates and thiazolidinediones, are used in the clinical treatment of dyslipidemia and diabetes. 

PPARgene database is not only a resource that curated the experimentally verified PPAR-α, -β/δ, and -γ target genes but also a platform that integrated tool(s) for predicting novel PPAR target genes.

Currently, users can

* search the experimentally verified PPAR target genes for each isoform;
* predict potential PPAR target genes using an established machine learning model;
* download the datasets of experimentally verified PPAR target genes;
* download the datasets of predicted PPAR target genes;


If you have any questions, please create an issue here: https://github.com/fangli0813/doc-PPARgene/issues

You can also email me: fangli9@mail.sysu.edu.cn

## Citation: 
[Li Fang, et al. PPARgene: A Database of Experimentally Verified and Computationally Predicted PPAR Target Genes. PPAR Research. 2016: 6042162](http://dx.doi.org/10.1155/2016/6042162)
